site/_docs/history.md

```diff
@@ -49,6 +49,8 @@ Guava versions 21.0 to 32.1.3-jre;
 other software versions as specified in gradle.properties.
 
 #### New features
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5855">CALCITE-5855</a>]
```

Comment:
This file is a template file, we don't need to change this. The release manager will modify it.